frontend/src/modules/search/withSearch.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 
 import * as React from 'react';
-import createMarker from 'react-content-marker';
+import mark from 'react-content-marker';
```

Comment:
`import { mark } from 'react-content-marker';`

Otherwise you're just importing the same thing (the default export) under a different name. :-)

Reply:
Of course :)